firmware/projects/Demo/CAN/Receive/main.cc

```diff
@@ -4,27 +4,24 @@
 #include <cstdint>
 
 #include "bindings.h"
-#include "generated/can/demobus_can_messages.h"
-#include "generated/can/demobus_msg_registry.h"
-#include "shared/comms/can/can_bus.h"
+#include "generated/can/demobus_bus.h"
+#include "generated/can/demobus_messages.h"
+#include "shared/comms/can/bus.h"
 
-generated::can::DemobusMsgRegistry veh_can_registry{};
+using namespace generated::can;
 
-shared::can::CanBus veh_can_bus{
-    bindings::veh_can_base,
-    veh_can_registry,
-};
+DemobusBus veh_can_bus{bindings::veh_can_base};
```

Comment:
All this demo bus and veh bus is a bit odd I think you should pick one or the other.

Reply:
good catch